manifests/01-role.yaml

```diff
@@ -30,3 +30,15 @@ rules:
   - rolebindings
   verbs:
   - "*"
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
Do we need all these permissions here?  If yes we can just put `"*"`

Reply:
I've been told (though I can't remember where) that best practice is to enumerate all the permissions rather than using `"*"`.

Reply:
Jason found this for me

https://source.redhat.com/groups/public/product-security/content/product_security_wiki/operators_secure_deployment_guide#4-1-avoid-wildcards

...though the example makes it somewhat ambiguous whether it's okay to use the wildcard when you _do_ need all the verbs.

Reply:
that sounds fair

Reply:
Also [from Grant](https://coreos.slack.com/archives/C0281LN70AC/p1643838336786099?thread_ts=1643836110.625409&cid=C0281LN70AC): "i think it's because verbs are largely opaque, the verbs that make up  `*` today might not be the same as `*` in the future"